Branch-TestBed/Branch-TestBed/AppDelegate.m

```diff
@@ -46,7 +46,12 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // Required. Initialize session. automaticallyDisplayDeepLinkController is optional (default is NO).
     [branch initSessionWithLaunchOptions:launchOptions automaticallyDisplayDeepLinkController:YES deepLinkHandler:^(NSDictionary *params, NSError *error) {
         if (!error) {
+
             NSLog(@"initSession succeeded with params: %@", params);
+            NSDictionary *searchAds = [BNCPreferenceHelper preferenceHelper].appleSearchAdDetails;
```

Comment:
We don't want people accessing our preference helper storage details so let's remove this prior to merge